Makefile

```diff
@@ -233,10 +221,10 @@ bin-systray-windows: ## Build `skywire-visor` and `skywire-cli` with systray sup
 bin-systray-windows-appveyor: ## Build `skywire-visor` and `skywire-cli` with systray support
 	powershell 'Get-ChildItem .\cmd | % { ${OPTS} go build -tags systray -o ./ $$_.FullName }'
 
-bin-systray: ## Build `skywire-visor`, `skywire-cli`
+bin-systray: ## Build `skywire-visor`, `skywire-cli` | In MacOS (darwin) should set CGO_ENABLED=1 for skywire-visor
 	CGO_ENABLED=0 ${OPTS} go build ${BUILD_OPTS} -tags systray -o ./ ./cmd/skywire-visor
```

Comment:
We can have `CGO_ENABLED=0` and `CGO_ENABLED=1` in a variable and check the OS like we are checking for windows `if ($(OS),Windows_NT)` and if the OS is mac then we set `CGO_ENABLED=1` .

Reply:
Fixed.